infoblox_client/utils.py

```diff
@@ -32,19 +31,27 @@ def is_valid_ip(ip):
     return True
 
 
+def get_macstr_from_mac(mac):
+    return ''.join(c for c in mac if c not in ':')
+
+
 def generate_duid(mac):
     """DUID is consisted of 10 hex numbers.
 
-    0x00 + 3 random hex + mac with 6 hex
+    0x00 + 3 hex from last 3 character of mac_str + mac with 6 hex
```

Comment:
Just curious - what's the reasoning behind using this strategy.
For example, why don't we just copy the, say, last 3 octet of the original MAC - which seems simpler.